packages/snaps-utils/src/structs.test.ts

```diff
@@ -157,7 +172,7 @@ describe('createFromStruct', () => {
   });
 
   it('throws the raw error if an unknown error is thrown', () => {
-    jest.spyOn(superstruct, 'create').mockImplementation(() => {
+    createMock.mockImplementationOnce(() => {
```

Comment:
Implemented new logic for mocking imported functions, as spying on default import no longer works with the following error:

```
Cannot use spyOn on a primitive value; undefined given
```